content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Visualizing Your Mesh
+description: This task shows you how to visualize your services within an Istio mesh.
+weight: 49
+keywords: [telemetry,visualization]
+---
+
+This task shows you how to visualize different aspects of your Istio mesh.
+
+As part of this task, you will install the [Kiali](https://www.kiali.io) add-on
+and use the web-based graphical user interface for viewing service graphs of
+the mesh and your Istio configuration objects. You will also use the Kiali
+Public API to generate graph data in the form of consumable JSON.
```

Comment:
I will replace "graph" with "diagram" - however, keep in mind, the Kiali UI uses the term "graph" in places such as "Graph Type" and the "Graph View". This is in relation to the topology graph pages of the GUI. Where the word "graph" relates directly to a component in the Kiali UI, I will leave the word "graph" in place to avoid confusion (e.g. there is a "Graph Type" drop down menu item in Kiali UI - I should not refer to this as "Diagram Type" since it would not match what the user sees in the UI).

Reply:
Also, keep in mind that these diagrams are really, technically, called "graphs" in that we use different graph models/algorithms implemented by Cytoscape (a third party library which also calls these things "graphs" - http://js.cytoscape.org/#notation/graph-model )

Reply:
Actually, now that I'm looking more closely at the terminology, I believe these should be called "graphs" - these have nodes/edges, are directed, and cyclic or acyclic depending on connections between nodes. These are all features of graphs (nomenclature of the technology in question). In other words, do not think of the word "graph" here as in a "line graph" or a "bar graph" where you have mathematical data represented visually - these are "graphs" in the sense of "directed acyclic graphs" or the like, which is what Kiali is dealing with.

Reply:
I understand if you consider, and based on the UI examples you mention, that the term graph is better suited, even if it is being used incorrectly. If you agree, my advice is to leave the term alone for now.